crates/context_aware_config/src/api/default_config/handlers.rs

```diff
@@ -184,3 +194,59 @@ async fn get(db_conn: DbConnection) -> superposition::Result<Json<Vec<DefaultCon
     let result: Vec<DefaultConfig> = default_configs.get_results(&mut conn)?;
     Ok(Json(result))
 }
+
+pub fn check_key_in_context_overrides(
```

Comment:
The function name suggest it just checks, but we are returning back list of context ids where the key is being used, either change the function name or the result value type.

Reply:
Maybe you can name it `getKeyUsageContextList `

Reply:
Ok, sure